modules/document_repository/php/document_repository.class.inc

```diff
@@ -142,28 +142,6 @@ class Document_Repository extends \NDB_Menu_Filter_Form
              "id"   => $id,
          );
     }
-    /**
```

Comment:
that took me way too long to process. thx for removing it :+1:

Reply:
No problem, I love deleting code